assets/extensions/update-notification/woocommerce-notice.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Multiple from './multiple';
+import UpdateAvailableLabel from './update-available-label';
+import { Col } from '../grid';
+import { UpdateIcon } from '../../icons';
+import ExtensionActions from '../extension-actions';
+
+/**
+ * WooCommerce notice component.
+ *
+ * @param {Object} props             Component props.
+ * @param {Array}  props.extensions  Extensions list.
+ * @param {Array}  props.isConnected Whether the site is connected to WC.com.
+ */
+const WooCommerceNotice = ( { extensions, isConnected } ) => {
+	const extensionsWithUpdate = extensions.filter(
+		( extension ) => extension.has_update
+	);
+
+	const updatesCount = extensionsWithUpdate.length;
+	const isInstalled = ! window.sensei_extensions?.installUrl;
+	const isActive = ! window.sensei_extensions?.activateUrl;
+
+	if ( 0 === updatesCount || ( isInstalled && isActive && isConnected ) ) {
+		return null;
+	}
+
+	const showConnectionNotice = ! isConnected && isInstalled && isActive;
+	const title =
```

Comment:
Maybe creating the title and the action with ifs would make it more readable. WDYT?
It would likely be more maintainable too if we need to change the conditions, for example.

Reply:
OK, I shall conform to the boring 'ol `if` statement 😄 - https://github.com/Automattic/sensei/pull/4215/commits/736b8df99768ea884e6212938c0e7cd5ffb1556c